pkg/agent/openflow/pipeline.go

```diff
@@ -1134,7 +1093,7 @@ func (f *featureService) flowsToTrace(dataplaneTag uint8,
 			if ovsMetersAreSupported {
 				fb = fb.Action().Meter(PacketInMeterIDTF)
 			}
-			fb = fb.Action().SendToController(uint8(PacketInReasonTF))
+			fb = fb.Action().SendToController(uint8(PacketInCategoryTF), PacketInNoMultiOperation, false)
```

Comment:
Here `PacketInNoMultiOperation` is meaningless for Service feature, right?

Reply:
Yes, removed it after changing to use []byte for SendToController.